pkg/scheduler/controller/common/seed.go

```diff
@@ -28,6 +28,10 @@ func VerifySeedReadiness(seed *gardencorev1beta1.Seed) bool {
 		return false
 	}
 
+	if cond := gardencorev1beta1helper.GetCondition(seed.Status.Conditions, gardencorev1beta1.SeedSystemComponentsHealthy); cond == nil || cond.Status != gardencorev1beta1.ConditionTrue {
+		return false
+	}
```

Comment:
Hm, I don't know whether the scheduler should look at this condition. My gut feeling is rather 'no' since this could temporarily flap. In such situation we wouldn't schedule new shoots...

Reply:
I think it makes sense to only look at this before triggering control plane migration, as there is a chance that you would break an already running shoot by moving it to a broken seed (even if it is only temporarily broken).